frontend/src/components/CompanyInput/CompanyInput.js

```diff
@@ -16,8 +16,12 @@ export default class CompanyInput extends Component {
         this.state = {
             companyName: ""
         }
+        this.handleClick = this.handleClick.bind(this)
     }
 
+    // route the user to the information component with the company name they input
+    handleClick (){ this.props.history.push("/information/"+this.state.companyName);}
+
     render() {
         return (
             <Form id="company-form">
```

Comment:
Add `onSubmit={this.handleSubmit}` to `<Form />` so that the input gets processed once the user hits enter on keyboard after typing in input. This provides 2 way for getting stock details: hit enter on keyboard after typing in input and clicking enter button.